### ISSUE TITLE
d

### ISSUE THREAD

**Issue by BenMH** [aradzie/keybr.com#347]
Is there any chance of adding an option to use a British dictionary, in addition to the British keyboard layout?

It would be nice to type the words spelled the way I am used to -- I wouldn't want to suddenly start typing realize instead of realise on the internet in general. I might be mistaken for a Yank! ;-)

**Comment by plastikfan** [comment]
Yeah I agree with this. It would be nice to be able to set a British dictionary. The problem is when we practice, we developing the wrong muscle memory, which will be difficult to undo.

**Comment by peterjc** [comment]
Duplicate of #65 (which has a more meaningful issue title too - whoops!)

**Comment by aradzie** [comment]
Fixed in #564
